model/task_lifecycle.go

```diff
@@ -862,7 +860,7 @@ func MarkEnd(ctx context.Context, settings *evergreen.Settings, t *task.Task, ca
 		if err != nil {
 			return errors.Wrap(err, "getting display task")
 		}
-		if err = checkResetDisplayTask(ctx, settings, dt); err != nil {
+		if err = checkResetDisplayTask(ctx, settings, requester, "", dt); err != nil {
```

Comment:
We should try to avoid ever passing through the empty string when we can avoid it. Should we just pass through the requester, since that's what the original checkResetDisplayTask logic was? (Also do we need to assign that to a variable?)

Reply:
The variable naming for this part is a little out of order actually -- this function already takes in a `caller` parameter, the `checkResetDisplayTask` function expects the first string param to be caller and the second to be origin, so I renamed the variable above here to be origin and not requester